index.html

```diff
@@ -4,7 +4,7 @@
 
 <h1>Pull request practice</h1>
 <p>
-  <a href="https://github.com/osscommunity/pr-practice">Add yourself!</a>
+  <a href="https://github.com/osscommunity/pr-practice/blob/gh-pages/README.md">Add yourself!</a>
```

Comment:
Well, since the link will be viewed at http://osscommunity.github.io/pr-practice/, it would actually need the full URL to see the rendered version. Otherwise, it would show http://osscommunity.github.io/pr-practice/README.md.